arrow-site/docs/docs/core/README.md

```diff
@@ -183,7 +183,7 @@ dependencies {
 
 #### Next development version
 
-If you want to try the latest features, replace `1.0.1` with one of the latest `alpha`, `beta` or `rc` publications.
+If you want to try the latest features, replace `1.1.0` with on of the latest `alpha`, `beta` or `rc` publications.
```

Comment:
[minor, typo] `one` -> `on`

Reply:
This appears in other places too, by the way.

Reply:
thanks for the catch 👍🏾